src/FilterService/FilterType/ElasticSearch/SelectCategory.php

```diff
@@ -74,6 +74,7 @@ public function addCondition(AbstractFilterDefinitionType $filterDefinition, Pro
         }
 
         $currentFilter[$filterDefinition->getField()] = $value;
+        $value = (string)$value;
```

Comment:
why not cast the value directly in the if or inside the trim function? 

right now, it doesn't make much difference, but if for what ever reason additional logic working with the value is added after the cast, it might result in unwanted behavior. 

WDYT?